src/Pages/Home.jsx

```diff
@@ -1,41 +1,66 @@
-import { React } from "react";
-
-import { Link } from "react-router-dom";
+import { React, useState } from "react";
+import { Link, useHistory } from "react-router-dom";
 import LoginButton from "../Components/LoginButton";
+
 const Home = () => {
+
+  const handleClick = (info) => {
+      window.location.href = `/${info}`;
```

Comment:
Yo!🧟‍♂️  Use <Link /> tag from react-router-dom for navigation. SPA's shouldn't load on routing.

###### Resource:
[React Router Link](https://reactrouter.com/web/api/Link)